### ISSUE TITLE
make it faster

### ISSUE THREAD

**Issue by RustJason** [php-build/php-build#301]
I think we should add options to let user add use define options such like -j when execute make

**Comment by rogeriopradoj** [comment]
Hi, @RustJason , do you have suggestions on how to do that?

/cc @php-build/collaborators @php-build/owners

**Comment by henriquemoody** [comment]
I have not much intimacy with the code itself, but I know rubyenv does something similar with environment variables, so we could have something (if we don't have already) like it:

```bash
export PHPBUILD_EXTRA_MAKE_ARGUMENTS="-j$(nproc)" && php-build ... 
```

**Comment by henriquemoody** [comment]
Fixed by #323
